src/CozyUtils.spec.js

```diff
@@ -241,6 +241,57 @@ describe('CozyUtils', () => {
       expect(result).toEqual(fakeAccount)
     })
 
+    it('should update an existing account with the same email', async () => {
+      const previousAccount = {
+        _id: '123',
+        _rev: 'abc',
+        canLinkContacts: true,
+        shouldSyncOrphan: true,
+        lastSync: null,
+        lastLocalSync: null,
+        name: 'john.doe@gmail.com',
+        _type: DOCTYPE_CONTACTS_ACCOUNT,
+        type: APP_NAME,
+        sourceAccount: 'previousAccountId',
+        version: 1
+      }
+      const findSpy = jest.fn().mockResolvedValueOnce({
+        data: []
+      })
+      findSpy.mockResolvedValueOnce({
+        data: [previousAccount]
+      })
+      cozyUtils.client.collection = jest.fn(() => ({
+        find: findSpy
+      }))
+      cozyUtils.client.save = jest.fn().mockResolvedValue({
+        data: {
+          id: 'saved-contact-account'
```

Comment:
I think that `id` should be `123` in this case, right?

Reply:
Yes, it's just mocked, but yes. You're right, I'll update this for clarity.